### ISSUE TITLE
Error export entries

### ISSUE THREAD

**Issue by alex-wsul** [fluentform/fluentform#458]
When I try export entries to a csv, the browser close conection with empty screen.
I realized that it occurs because a large number of fields (200 fields) in the form.
When I select a fewer number of fields the export run ok
Maybe the number of variables on the GET on browser excced  the limit, but no show errors...

**Comment by HrDelwar** [comment]
@alex-wsul You’re right—the issue is caused by the large number of fields in the form. It seems that the number of variables in the GET request exceeds the limit, leading the browser to disconnect without displaying any errors. We appreciate you bringing this to our attention, and we will address it in our next update. Thank you for your feedback!

**Comment by HrDelwar** [comment]
@alex-wsul This has been fixed in version 5.2.4. Please update to the latest version to use the improved export functionality.
